workflows/proteomics/clinicalmp/clinicalmp-database-generation/iwc-clinicalmp-database-generation-tests.yml

```diff
@@ -0,0 +1,48 @@
+- doc: Test outline for iwc-clinicalmp-database-generation
+  job:
+    HUMAN-SwissProt-Protein-Database:
```

Comment:
Can you adjust all of the input and output labels to be more "human friendly" ? You can use a normal whitespace here, no need for `-` or `_`. These labels are shown in the invocation component and the workflow report, so it's nice if they look good.

Generally speaking you also don't need to include the filetype extension in the label, the right filetype is shown in the user interface.

Reply:
You've marked this as reolved but it looks like you didn't actually do this ?

Reply:
somehow when i remove the dashes my tests fail

Reply:
You have to change them in the workflow too.

Reply:
yeah, i did that too..still it would fail. Not sure why

Reply:
I can do this for now, if you do a find and replace with something like vscode or sed this will work. If you're changing this by hand it is easy to make a mistake. I guess the takeaway is to directly give it instructive labels, then the autogenerated tests will match those labels.

Reply:
And you can also check this with `planemo workflow_lint <workflow_directory>`}:
```
.. WARNING: Input HUMAN-SwissProt-Protein-Database of workflow step HUMAN-SwissProt-Protein-Database is disconnected.
.. WARNING: Input Species_UniProt_FASTA of workflow step Species_UniProt_FASTA is disconnected.
.. WARNING: Input Contaminants-cRAP-Protein-Database of workflow step Contaminants-cRAP-Protein-Database is disconnected.
.. WARNING: Input input_type of workflow step Generating Customized Database is disconnected.
.. CHECK: Tests appear structurally correct for ./iwc-clinicalmp-database-generation.ga
.. CHECK: All tool ids appear to be valid.
```
is the current state

Reply:
Thank you! I think I fixed this!